### PR TITLE
Adding heritage patron

### PR DESCRIPTION
**Pull request by lufercc**
Adding the Classes to Change to heritage model

### CODE REVIEW

File: src/com/jalasoft/search/model/Asset.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * @version  1.0
+ * @author Luis Cachi
+ */
+public class Asset {
+    private int size;
+    private String name;
+    private String path;
+    private String owner;
+
```
Comment (Anaeli) [inline src/com/jalasoft/search/model/Asset.java]: Add comment

File: src/com/jalasoft/search/model/File.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.jalasoft.search.model;
+
+/**
+ *This class implements the File for model a Folder object.
+ * @version  1.0
+ * @author Luis Cachi
+ */
+public class File extends Asset {
+    String extension;
+
```
Comment (Anaeli) [inline src/com/jalasoft/search/model/File.java:10]: Add comment , are you sure that this way is maintainable due you only be able to create a file with this information, what happen with other information about file?

File: src/com/jalasoft/search/model/Folder.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.jalasoft.search.model;
+
+/**
+ * This class implements the Folder for model a Folder object.
+ * @version  1.0
+ * @author Luis Cachi
+ */
+public class Folder extends Asset{
+    int cantOfFiles;
+    public Folder(String name, int size, String path, String owner, int cantOfFiles) {
```
Comment (Anaeli) [inline src/com/jalasoft/search/model/Folder.java]: add comment

File: src/com/jalasoft/search/model/File.java (hunk 2 above)
Comment (lufercc) [reply src/com/jalasoft/search/model/File.java:10]: about the unique difference attribute
I will add more according to the projects will increase

**Pull request merged**